encryption/c#-example/Program.cs

```diff
@@ -13,6 +13,7 @@ public static void Main(string[] args)
         {
           
             string plaintext = "Hello World";
+            //key must be 16bytes

```

Comment:
btw rapiin komeng nya dong klo boleh
```suggestion
            // key must be 16 bytes
```